Gujarati/gujarati.md

```diff
@@ -1,6 +1,5 @@
-When object to be gained is sufficiently valued, 
-and efforts towards its attainment are persistently 
-followed without intermission, then the steadiness of mind is secured.
+When object to be gained is sufficiently valued, and efforts towards its attainment are persistently followed without intermission, then the steadiness of mind is secured.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.